DemoServer/AdditionalLanguages/go/StaticIndexes/staticIndexesOverview/metadata.json

```diff
@@ -59,13 +59,16 @@
                 "",
                 "* _Additional indexing options, per index-field, that can be defined inside the Map function are_:<br>",
                 "   Index spatial data<br>",
+                "   Index counters data<br>",
                 "   Index dynamic fields<br>",
+                "   Index time-series data<br>",
                 "   Index hierarchical data<br>",
                 "   Index document fields as json<br>",
                 "   Boosting the indexed field value<br>",
-                "   Index counters & attachments names<br>",
                 "   Index data from a related document<br>",
+                "   Index attachments details & content<br>",
                 "   Index data from the document's metadata<br>",
+                "   Use LINQ extensions, such as: Reverse, ParseInt, and more",
```

Comment:
No objections here, but I noticed something else which could be corrected

Up above on line 14, "Defined with JavaScript"
Since we are not withholding mentioning LINQ here, we could mention it there as well, since all other language-specific texts are stating "Defined with LINQ or JavaScript"

What do you think?

Reply:
@DejanMilicic I clicked rereview as looks like there are some items pending to avoid accidental merge

Reply:
done